geniza/corpus/templates/corpus/snippets/document_transcription.html

```diff
@@ -1,12 +1,18 @@
-{% load i18n corpus_extras %}
+{% load i18n static corpus_extras %}
 <section id="itt-panel">
     {# NOTE: inputs must be kept as SIBLINGS of panel-container for CSS selection/controls #}
     {# images displayed by default; disable if no images are available #}
-    <input type="checkbox" id="images-on"  {% if document.has_image %}checked="true" {% else %} disabled="true"{% endif %}/><label for="images-on">show images</label>
+    {# translators: label for checkbox toggle to show the images panel for a document #}
+    <input type="checkbox" id="images-on" aria-label="{% translate 'show images' %}" {% if document.has_image %}checked="true" {% else %} disabled="true"{% endif %}/>
+    <label for="images-on"></label>
     {# transcription displayed by default; disable if no content is available #}
-    <input type="checkbox" id="transcription-on" {% if document.digital_editions %}checked="true" {% else %} disabled="true"{% endif %}><label for="transcription-on">show transcription</label>
+    {# translators: label for checkbox toggle to show the transcription panel for a document #}
+    <input type="checkbox" id="transcription-on" aria-label="{% translate 'show transcription' %}" {% if document.digital_editions %}checked="true" {% else %} disabled="true"{% endif %}/>
+    <label for="transcription-on"><svg><use xlink:href="{% static 'img/ui/all/all/transcription-toggle.svg' %}#transcription-toggle" /></svg></label>
```

Comment:
why this `svg` / `use` notation with id? is this so you can style with css?

Reply:
Yes! It's for setting the "disabled" color on the SVGs. It could also be done with background images and separate files for "disabled", as I did with the PNGs.